.gitignore

```diff
@@ -0,0 +1,212 @@
+# Created by .ignore support plugin (hsz.mobi)
```

Comment:
Can we please clean this file up? It contains a lot of overlapping and unrelated entries.

Reply:
its a generated file, I would either go with minimal or accept it.
but not too worried, note that this part of the PR is superseded by #2